blocks/library/button/index.js

```diff
@@ -80,7 +80,7 @@ registerBlockType( 'core/button', {
 		const { url, text, title, align = 'none' } = attributes;
 
 		return (
-			<div className={ `align${ align }` }>
```

Comment:
I would suggest `blocks-button` instead to avoid ambiguity with already defined `button` classes.